### PR TITLE
Dev drew

### PR DESCRIPTION
**Pull request by zhiyuli**

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import logging
 
-logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+#logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
 logger = logging.getLogger(__name__)
```
Comment (cuahsimarko) [inline settings.py]: can you start to incorporate/merge your logging config into this one? Eventually, this will be the only logging config

Comment (cuahsimarko) [reply settings.py]: format="%(asctime)s [%(levelname)-5.5s]  %(message)s", as well as the handlers

File: czo2hs.py [hunk 2]
```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
...
     hs = CZO_HS_Account_Obj.get_hs_by_czo("default")
     hs_id = hs.createResource("CompositeResource",
-                              "czo2hs migration log files {}".format(start_time),
+                              "czo2hs migration log files {}".format(start_time_str),
                               )
-    file_id = hs.addResourceFile(hs_id, os.path.join(logdir, log_file_name))
-    file_id = hs.addResourceFile(hs_id, results_file)
+    file_id = hs.addResourceFile(hs_id, log_file_path)
+    file_id = hs.addResourceFile(hs_id, czo_hs_id_lookup_csv_path)
     print("Log files uploaded to HS res at {}".format(hs_id))
```
Comment (cuahsimarko) [inline czo2hs.py]: let's move LOGDIR and make it a constant; move it to settings.py

File: czo2hs.py [hunk 3]
```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
...
+    czo_hs_id_lookup_df.to_csv(czo_hs_id_lookup_csv_path, encoding='utf-8', index=False)
 
     # upload log file and results file to hydroshare
-    logging.info("*"*20 + "Uploading log file to HS" + "*"*20)
+    logging.info(text_emphasis("Uploading log file to HS"))
     hs = CZO_HS_Account_Obj.get_hs_by_czo("default")
     hs_id = hs.createResource("CompositeResource",
-                              "czo2hs migration log files {}".format(start_time),
+                              "czo2hs migration log files {}".format(start_time_str),
                               )
```
Comment (cuahsimarko) [inline czo2hs.py]: don't create a new variable for start_time_str, just str(start_time) here in line 170

File: czo2hs.py [hunk 4]
```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
                                                                        error_item["error_msg_list"]])))
     logging.info(text_emphasis("CZO_ID <---> HS_ID Lookup Table"))
     logging.info(czo_hs_id_lookup_df.to_string())
-    results_file = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time))
-    logging.info("*" * 20 + "Saving Lookup Table to {}".format(results_file) + "*" * 20)
+    czo_hs_id_lookup_csv_path = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time_str))
+    logging.info(text_emphasis("Saving Lookup Table to {}".format(czo_hs_id_lookup_csv_path)))
 
-    czo_hs_id_lookup_df.to_csv(results_file, encoding='utf-8', index=False)
+    czo_hs_id_lookup_df.to_csv(czo_hs_id_lookup_csv_path, encoding='utf-8', index=False)
```
Comment (cuahsimarko) [inline czo2hs.py]: shorter, friendlier names, please. czo hs csv are kind of known. What's specifically unique here?

File: czo2hs.py [hunk 5]
```diff
@@ -60,18 +60,20 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
...
-        log_progress(progress_dict, "Progress Report")
+    log_progress(progress_dict, "Progress Report", start_time=start_time)
 
 
 if __name__ == "__main__":
+
     start_time = dt.utcnow()
+    start_time_str = start_time.strftime("%Y-%m-%d_%H-%M-%S")
     logdir = "./logs"
-    log_file_name = "log_{}.log".format(start_time.strftime("%Y-%m-%d_%H-%M-%S"))
```
Comment (cuahsimarko) [inline czo2hs.py]: LOGDIR to settings.py

Comment (cuahsimarko) [reply czo2hs.py]: Make LOGDIR CAPS LETTERS and move to settings.py

File: czo2hs.py (hunk 4 above)
Comment (cuahsimarko) [reply czo2hs.py]: rename czo_hs_id_lookup_csv_path to results_file it's more readable

File: czo2hs.py (hunk 3 above)
Comment (cuahsimarko) [reply czo2hs.py]: don't include data types in variable names. rename this from start_time_str to start_time

File: czo2hs.py (hunk 2 above)
Comment (cuahsimarko) [reply czo2hs.py]: czo_hs_id_lookup_csv_path is too long. shorter, more concise variable names, please

Comment (cuahsimarko) [reply czo2hs.py]: change name back to results_file

**Comment by cuahsimarko** [comment]
Still some outstanding items

File: czo2hs.py (hunk 2 above)
Comment (zhiyuli) [reply czo2hs.py]: done in new commit (not sure why github is not updating)

File: czo2hs.py (hunk 3 above)
Comment (zhiyuli) [reply czo2hs.py]: fixed in new commit. 
I want to specify the formatting of the datetime str as it is used in filename which should not contain ":" or spaces

File: czo2hs.py (hunk 4 above)
Comment (zhiyuli) [reply czo2hs.py]: fixed in new commit

**Comment by cuahsimarko** [comment]
Merge whenever you're ready

**Comment by zhiyuli** [comment]
Fixed most of the comments. But I dont how to move all logging configs into settings.py as the log file name still depends on the "start_time" in the __main__ block. Please take care of this during your refactoring.

**Pull request merged**